functionary/ui/templates/builder/build_detail.html

```diff
@@ -3,61 +3,66 @@
     <div id="build_detail"
          hx-swap-oob="true"
          {% if not completed %} hx-get="{% url 'ui:build-detail' build.id %}?poll=true" hx-trigger="every 15s"{% endif %}>
-        <nav aria-label="breadcrumb">
-            <ol class="breadcrumb">
+        <nav aria-label="breadcrumb mb-1">
```

Comment:
The margin between the breadcrumb and the `<hr>` is different on the detail page versus the list page, which results in a subtle shift as you navigate between pages.  Here's a snippet from the two jammed together with the text aligned.  You can see slightly bigger gap between the text and the `<hr>` on the left.

![image](https://user-images.githubusercontent.com/86254668/228697228-772fa663-4b54-4d69-bbe4-103cad82e53e.png)